### ISSUE TITLE
Publish on `conda-forge`

### ISSUE THREAD

**Issue by bstadlbauer** [buildkite/test-collector-python#18]
Hi!

Would anyone mind if I publish the test collector on [conda-forge](https://conda-forge.org/#page-top)? This way, folks that use `conda` to manage Python environments would have a native way to install the test-collector.

This would entail creating a new "feedstock" repository (here is an [example](https://github.com/conda-forge/pip-feedstock)) which builds the `conda` package based on the package available on PyPi.

Happy to set things up and maintain the feedstock (I already maintain e.g. [this feedstock](https://github.com/conda-forge/flytekitplugins-dask-feedstock)). In case you'd like to, I can also add some of the maintainers here as feedstock maintainers to increase the bus factor. There is usually not much maintenance to do (unless dependencies change)

**Comment by swebb** [comment]
Hi @bstadlbauer. Sorry for the slow response. I'm trying to get an answer for you.

**Comment by meiqimichelle** [comment]
Hi @bstadlbauer! Apologies that this took so long to respond to, and appreciate your patience!


> Would anyone mind if I publish the test collector on [conda-forge](https://conda-forge.org/#page-top)?

We would not mind at all! In addition, the code is [MIT-licensed](https://github.com/buildkite/test-collector-python/blob/main/LICENSE.txt), so you are free to distribute as you see fit 😊


> Happy to set things up and maintain the feedstock (I already maintain e.g. [this feedstock](https://github.com/conda-forge/flytekitplugins-dask-feedstock)). In case you'd like to, I can also add some of the maintainers here as feedstock maintainers to increase the bus factor.

Thank you very much, and yes, we'd appreciate this! 🙇🏻‍♀️

**Comment by bstadlbauer** [comment]
@meiqimichelle No worries, I know how it goes!

Would you happen to have a recommendation on which maintainer to add from your side?

**Comment by meiqimichelle** [comment]
Ah, I think I mis-read that sentence 🙃. I think I read it as adding more folks on your side. I think for right now its best to unblock you to move ahead individually, and we can sort out bus-factor later (and potentially add folks from our side later, but not now, if that makes sense!).

**Comment by bstadlbauer** [comment]
Completed, the package is now available on `conda-forge` ([feedstock](https://github.com/conda-forge/buildkite-test-collector-feedstock), [package](https://anaconda.org/conda-forge/buildkite-test-collector)). 

I've also opened https://github.com/buildkite/test-collector-python/pull/22 in case you'd like to add that to your docs, feel free to close though.

**Comment by meiqimichelle** [comment]
Thank you @bstadlbauer for doing this, and opening the docs issue. Hope it helps your workflows to have this in `conda-forge` as well!

_I love a good full-circle, complete with a docs PR_ 💚🎉 Today's gonna be a good day 😊
